packages/docs/content/04-clusters/3-cluster-updates.md

```diff
@@ -0,0 +1,77 @@
+|Add Pack|Add a new pack to the cluster profile    |New Kibana 7.2.4 layer is added|
+|Delete Pack|Delete the existing pack from the cluster profile      |Kibana 7.2.4 layer is deleted|
+|Attach Pack Manifest|Delete the existing pack from the cluster profile      |Manifest security is attached to the pack Kubernetes|
+|Update Pack Manifest|The attached pack manifest content is updated in the cluster profile|manifest security is updated in the pack Kubernetes|
+|Delete Pack Manifest |The attached pack manifest is deleted from the cluster profile|manifest security is deleted in the pack Kubernetes|
+ 
+ 
+## ‘Available Updates’- pop up screen
+The ‘**Available Updates**’ pop-up carries details on **‘current cluster values and incoming profile changes’**. As the tag suggests, the left side carries the current values in the cluster in view only mode and the right side carries the updated values which are in editable format. Once the ‘confirm updates’ is selected the right side updates get overridden to the left side and the cluster profile gets updated . While adding and deleting a cluster feature ‘current cluster values and incoming profile changes’ will not be there, only current cluster values window in editable format will only be present.
+ 
+Related to ‘current cluster values and incoming profile changes’ the contents of this window takes two forms:
+* current cluster values / incoming profile changes:
+If a pack contains no overridden values, an edit to the pack values displays current cluster values / incoming profile changes format .
+* incoming profile changes / current cluster values:
+If a pack contains already overridden values and a further edit to the pack values displays incoming profile changes / current cluster values.
+ 
+**Note:**
+Once a change is made at the cluster profile and notification update is not applied to the cluster (pending notification), then if the user decides to revert the changes at cluster profile then the pending notification on the cluster gets cleared automatically.
+
+Eg: new pack added - update notification generated  - keep the notification pending - remove the added pack - No updates visible on the cluster.
+ 
+# Cluster Profile Notification
+ 
+The registry updates such as new pack or helm chart release, existing pack values update generates the notification on the cluster profiles. If the user selects the Kubernetes pack with smart tag (eg: version 1.20.x) while adding the pack, any new Kubernetes release available at the registry auto updates the cluster profiles to update to the latest Kubernetes. Instead if the user selects a specific version (eg: version 1.20.2) of the pack then users need to update to the new version by updating the cluster profile manually. Any update to the cluster profile triggers the notification on the clusters and users should confirm the updates in order to apply the changes to the running kubernetes cluster.
```

Comment:
kubernetes or Kubernetes. We can have a standard convention. Kubernetes everywhere would be good